rust/kcl-wasm-lib/src/api.rs

```diff
@@ -0,0 +1,80 @@
+use gloo_utils::format::JsValueSerdeExt;
+use kcl_api::{Error, File, FileId, LifecycleApi, ProjectId};
+use wasm_bindgen::prelude::*;
+
+use crate::Context;
+
+#[wasm_bindgen]
+impl Context {
+    #[wasm_bindgen]
+    pub async fn open_project(&self, project: usize, files: &str, open_file: usize) -> Result<JsValue, JsValue> {
+        console_error_panic_hook::set_once();
+
+        let files: Vec<File> =
+            serde_json::from_str(files).map_err(|e| JsValue::from_serde(&Error::deserialize("file", e)).unwrap())?;
```

Comment:
We shouldn't be calling `unwrap()` in this file.

Reply:
This was copy and pasted from another usage in a sibling module and it's only unwrapping the result of serialising an error, which I hope should never fail (I'm not sure how to handle this without unwrapping either - I guess we could send a dummy JsValue, but in that case the frontend might choke on it when expecting an error?)